smqtk_iqr/web/iqr_service/iqr_server.py

```diff
@@ -166,7 +168,7 @@ def get_default_config(cls):
                     "neighbor_index":
                         make_default_config(NearestNeighborsIndex.get_impls()),
                     "classifier_config":
-                        make_default_config(Classifier.get_impls()),
+                        make_default_config(ClassifyImage.get_impls()),
```

Comment:
Remember that the direct parallel to the `Classifier` interface is the now `ClassifyDescriptor` interface.